### DISCUSSION TITLE
Indicator for "last active" or "last online" and/or "available to chat"

### DISCUSSION THREAD

**Discussion by aklidata** [PullDating/Org#79] [Ideas & Features]
Activity Flag, add a flag to profiles for last active/add availability to chat.

**Comment by WilliamDormer** [comment]
Yes, there should be a way to tell if someone is online on the app or not.

**Comment by Oshyan** [comment]
Not just are they online/active now, but a literal date (or at least date *range*) for when they last logged-in to the app. E.g. "This user was last online 1 month ago", "3 months ago", etc. This also relates to automated deactivation, which I think is important for transparency, and focusing on maximizing the benefits of user time and interaction (or put another way, **not wasting people's time**, which I feel a lot of dating apps do!).

**Comment by ajbott** [comment]
Activity flags would be really interesting to have. I think users should also have the ability to set custom activity flags just like in Microsoft Teams/Discord.
